week-1-ts-ecosystem/w01e01-typescript-basics/src/main.ts

```diff
@@ -16,15 +18,8 @@ const user: any = {
   },
 }
 
-function hasAddress(user: any): any {
-  return Boolean(user.address)
-}
-
-function hasGivenAge(requiredAge: any): any {
-  return (user: any): any => user.age >= requiredAge
-}
-
 const isAdult = hasGivenAge(18)
```

Comment:
Jeśli przeniosłeś metody wyżej to i tutaj to `isAdult` powinno moim zdaniem (konsekwentnie) wylądować w `userLogic.ts`